internal/cmdinit/cmdinit.go

```diff
@@ -126,61 +121,5 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 		}
 	}
 
-	if _, err = os.Stat(filepath.Join(args[0], man.ManFilename)); os.IsNotExist(err) {
-		fmt.Fprintf(c.OutOrStdout(), "writing %s\n", filepath.Join(args[0], man.ManFilename))
-		buff := &bytes.Buffer{}
-		t, err := template.New("man").Parse(manTemplate)
-		if err != nil {
-			return err
-		}
-
-		err = t.Execute(buff, r)
-		if err != nil {
-			return err
-		}
-
-		// Replace single quotes with backticks.
-		content := strings.ReplaceAll(buff.String(), "'", "`")
-
-		err = ioutil.WriteFile(filepath.Join(args[0], man.ManFilename), []byte(content), 0600)
-		if err != nil {
-			return err
-		}
-	}
-
 	return nil
 }
-
-// manTemplate is the content for the automatically generated README.md file.
-// It uses ' instead of ` since golang doesn't allow using ` in a raw string
-// literal. We do a replace on the content before printing.
-var manTemplate = `# {{.Name}}
```

Comment:
Are we removing the autogenerated README file?

Reply:
Are we using this ? The schema used by this functionality no longer exists in new types.go.

Reply:
I'm not sure what this has to do with the schema. As far as I can tell, this is just the template we use to generate the README.md file with the `kpt pkg init` command.

Reply:
Added Man generation back. Added Man field to `PackageInfo` type.